src/js/covidMap.js

```diff
@@ -64,8 +87,29 @@ fluid.covidMap.extractPostcodes = function (rows, field) {
     return postcodeHash;
 };
 
+// Apply special grade component for the mobile view
+fluid.covidMap.isMobileView = function () {
+    const mediaMatchResult = window.matchMedia("screen and (min-width: 1024px)");
+    return !mediaMatchResult.matches;
+};
+
+fluid.contextAware.makeChecks({
+    "fluid.isMobileView": "fluid.covidMap.isMobileView"
+});
+
 fluid.defaults("fluid.covidMap.map", {
-    gradeNames: ["hortis.leafletMap", "hortis.streetmapTiles", "hortis.CSVLeafletMap", "hortis.conditionalTemplateRenderer"],
+    gradeNames: ["hortis.leafletMap", "hortis.streetmapTiles", "hortis.CSVLeafletMap", "hortis.conditionalTemplateRenderer", "fluid.contextAware"],
+    contextAwareness: {
```

Comment:
You can see the test case showing how this facility works at https://github.com/fluid-project/infusion/pull/1038/files#diff-6972ab970b7ccb5a497ebc0fc51a19df59946de2ba24f58b1577c4ae40546f8cR1005-R1036

Reply:
Updated comment above to read "dynamicComponents" rather than "components" - my mind is already in the future when there will only be "regular effing components" : P